syntax_and_semantics/return_types.md

```diff
@@ -20,11 +20,13 @@ The return type follows the [type grammar](type_grammar.html).
 Marking a method as returning `Nil` will make it return `nil` regardless of what it actually returns:
 
 ```crystal
```

Comment:
This change seems a bit unrelated to the goal of the PR and issue #78. What's the rationale for introducing this change here?